CHANGES.txt

```diff
@@ -4,8 +4,13 @@ CHANGES
 0.15 (unreleased)
 =================
 
+- compatibility for python 3
+- update code to work with python2 and python3
+- update doctests to work with python2 and python3
+- adjust egg to work with newer version of setuptools
```

Comment:
I think "compatibility for Python 3" implies that code and doctests were updated to work with it.